articles/appliance/admin/updating-the-appliance.md

```diff
@@ -5,8 +5,58 @@ description: How to update the Appliance
 
 # Updating the Appliance
 
-Auth0 requires regular updates to your Appliance instances to ensure that you always have the latest features and improvements. The update interval is typically once per month.
+To ensure that your Appliance has the latest functionality, security, and bug fixes, Auth0 requires you to perform regular updates. While the Auth0 engineering team releases updates on a monthly basis, you should plan updates on a monthly, bi-monthly, and quarterly basis. You should not exceed **90 days** without updating the Appliance.
 
-During the update, the Appliance instances will need limited connectivity to the Internet to download the application and operating system updates). The update will be [started via the Management Dashboard](/appliance/dashboard/updates) and typically takes between 10 to 30 minutes.
+::: panel Appliance Releases
+For more information about Appliance Releases, please see the [Change Log](https://auth0.com/changelog/appliance).
+:::
 
-The update will be scheduled so that an Auth0 Customer Success Engineer is on the call with your operations team during the entire process.
+## Best Practices
+
+To ensure the update process goes as smoothly as possible, we recommend following the best practices detailed below.
+
+### Prior to the Update
+
+* Take a VM snapshot of each node you're updating.
+* Ensure that you've enabled access to your nodes for the Auth0 Customer Success Engineer who will be assisting you with the update.
+* Complete the pre-check test **one hour** prior the start of the update.
+  * Please be sure that your infrastructure engineer has administrative access to the Dashboard (specifically the Root Tenant Authority).
+  * Ensure that all [Health Checks](/appliance/dashboard/troubleshoot#health-check) are okay.
+
+  ::: panel Monitoring
+  For additional information on gathering testing information, please see [Appliance Monitoring](/appliance/monitoring).
+  :::
```

Comment:
I think you should remove the indentation, this doesn't looks very nice.

![image](https://cloud.githubusercontent.com/assets/11715799/26451092/306bc41c-4163-11e7-98ae-9f765bf214cc.png)